lib/commands/config.js

```
@@ -29,7 +31,19 @@ function load(configFile) {
         conventionConfig.commitTypes = conventionOverride.commitTypes || conventionConfig.commitTypes;
         conventionConfig.featureCommitTypes = conventionOverride.featureCommitTypes || conventionConfig.featureCommitTypes;
         conventionConfig.commitScopes = conventionOverride.commitScopes || conventionConfig.commitScopes;
-        conventionConfig.msgRegex = conventionOverride.msgRegex || conventionConfig.msgRegex;
+
+        if (hasMandatoryCaptureGroups(conventionOverride.msgRegex, ['type', 'scope', 'breaking', 'description'])) {
+            conventionConfig.msgRegex = conventionOverride.msgRegex;
+        } else {
+            console.warn(`[WARN] The message RegEx: ${conventionOverride.msgRegex} does not have the mandatory capture groups type, scope, breaking and description. The fallback is used`);
```

I would prefer to throw an error